libs/domain/search/src/services/suggestion/renderer/suggestion-renderer.service.ts

```diff
@@ -21,6 +23,7 @@ export type SuggestionRendererOptions = {
 export interface SuggestionRendererParams extends SuggestionFieldOptions {
   query: string;
   title: string;
+  icon?: string;
```

Comment:
We don't need duplicate options here we extend it `SuggestionRendererParams extends SuggestionFieldOptions`